hazelcast-client/src/test/java/com/hazelcast/client/impl/protocol/ClientExceptionFactoryTest.java

```diff
@@ -187,7 +176,6 @@ private boolean stackTraceArrayEquals(StackTraceElement[] stackTrace1, StackTrac
                 new Object[]{new EntryProcessorException(randomString())},
                 new Object[]{new ArrayIndexOutOfBoundsException(randomString())},
                 new Object[]{new ArrayStoreException(randomString())},
-                new Object[]{new AuthenticationException(randomString())},
```

Comment:
why did you remove?

Reply:
On 3.12, because we had two different modules, we are throwing `StubAuthenticationException` on the member and it is recreated as `AuthenticationException` on the client-side. 
So this test normally should not be passing. When I refactored the test to a correct state, this issue became apparent, and tests start to fail for `AuthenticationException`. 
I did not want to modify the test again for the `AuthenticationException` only. 
Now thinking about it, I can put a special test for `AuthenticationException` on `ClientProtocolErrorCodesTest`.